pkg/model/components/kubelet.go

```diff
@@ -81,11 +81,15 @@ func (b *KubeletOptionsBuilder) BuildOptions(o interface{}) error {
 	}
 
 	if clusterSpec.Kubelet.ClusterDNS == "" {
-		ip, err := WellKnownServiceIP(clusterSpec, 10)
-		if err != nil {
-			return err
+		if clusterSpec.KubeProxy.ProxyMode == "ipvs" && !clusterSpec.KubeDNS.NodeLocalDNS.Disable {
```

Comment:
The travis tests are reporting a panic here, you probably need to do additional `!= nil` checks

Reply:
Yeah need to fix that.